### DISCUSSION TITLE
Andrey Sitnik - Postcss, browserslist, nanostores, and more

### DISCUSSION THREAD

**Discussion by just-be-dev** [devtools-fm/devtools.fm#177] [Announcements]
## Background

- https://github.com/ai
- https://sitnik.ru/en/
- https://twitter.com/sitnikcode

## Intro

Andrew: Hello and welcome to the devtools.fm podcast. This is a podcast about developer tools and the people who make them. I'm Andrew and this is my co-host Justin.

Justin: Hey everyone! I'm excited to welcome our guest Andrey Sitnik. Andrey works as an engineer at Evil Martians (checkout ep 56 for more info about them) and is a powerful force in open source producing tools such as autoprefixer, postcss, and browserslist. Andrey, it's such a pleasure to have you on. Before we get started, would you like to tell our guests a bit more about yourself?

### Background

- [x] You're prolific in open source. How did you get started?
- [x] Where did you passion for maintaining and developing quality dev tools come from?

#### Browserslist

- [x] What is browsers list? Why did you create it?
- [x] With evergreen browsers does something like browserlist become less useful or will there ways be the need?

#### PostCSS

- [x] What are the origins of PostCSS? What was your original goal with the project?
- [x] What are some of your favorite plugins you've created?
- [x] PostCSS has become the go to tool in the space. Are there any plans for a "modern" version? Do you think tools like lightning CSS might come to replace PostCSS?

#### Oklch

- [x] What is OKLCH? Why is it better than other color solution?
- [x] Is okclh better for developers? Is it better for design systems?
- [x] Can I just start using oklch? Do other tools you've created help me start using it sooner?

#### Logux

- [x] What is logux?

### Reflections

- [x] What tool are you most proud of creating?
- [x] What's one of your lesser known tools that you think deserves more attention?

### Commercial Open Source

- [x] What COSS efforts have you been apart of at Evil martians?
- [x] Do you feel that COSS is the best way to sustain OSS or just a tool in a belt?

**Comment by hipstersmoothie** [comment]
https://github.com/comfyanonymous/ComfyUI
https://www.figma.com/dev-mode/

**Comment by hipstersmoothie** [comment]
@ai's

[https://oklch.com](https://t.co/QCsyusEV87)
[https://solberg.is/tailwind-with-oklch…](https://t.co/9oXE2Ft9Pn) by Jökull Solberg
[https://color-playground.ardov.me](https://t.co/coOIOETTbG) by Alexey Ardov
[https://huetone.ardov.me](https://t.co/qeM2FyfHXA) by Alexey Ardov
[https://myndex.com/APCA/](https://t.co/XBmM4mHQ76) by Myndex
[https://git.apcacontrast.com/documentation/WhyAPCA](https://t.co/bszS4dSGbs)

**Comment by just-be-dev** [comment]
tooltips:
- https://cooklang.org/
- https://github.com/maaslalani/invoice
